templates/components/products/product-view.html

```diff
@@ -230,6 +230,7 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
             {{/if}}
         </div>
         {{> components/common/share}}
+        {{> components/common/facebook-like-button}}
```

Comment:
Is there a reason why we can't pull this component inside `{{> components/common/share}}`

Reply:
I thought it would be better to separate it out since everything else is a svg instead of an iframe and they are about sharing whereas this is about liking but no real reason.

Reply:
@bookernath do you have an opinion on this one ? If now this PR is good to go.

Reply:
I like the idea of putting this new component _inside_ the share component.

If I were a theme developer that wanted to move my share icons around on the page, I'd probably expect Facebook to come along for the ride, so I think that would be more convenient.

Reply:
Made the requested changes @bookernath